README.md

```diff
@@ -227,6 +227,9 @@ The repository has the following permanent branches:
 
  * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch. This is the branch that is used in the deployed version of the app on Heroku.
 
+ * **bit** This branch is for MS-backend version specific to [BridgeInTech](https://github.com/anitab-org/mentorship-backend/tree/bit) project. All the contributing PRs related to BIT-MS integration issue must be sent to this branch.
```

Comment:
> [BridgeInTech](https://github.com/anitab-org/mentorship-backend/tree/bit) project

@The-Keshav-Agarwal, please change this link to the one given on @vj-codes post [here](https://github.com/anitab-org/mentorship-backend/pull/1036#discussion_r588648908)

Reply:
@mtreacy002 I replaced the link with https://github.com/anitab-org/bridge-in-tech-backend. please review it again.